fe/fe-core/src/test/java/org/apache/doris/nereids/jobs/cascades/DeriveStatsJobTest.java

```diff
@@ -70,7 +70,7 @@ public void testExecute() throws Exception {
         }
         Statistics statistics = cascadesContext.getMemo().getRoot().getStatistics();
         Assertions.assertNotNull(statistics);
-        Assertions.assertTrue(Precision.equals(0.5, statistics.getRowCount(), 0.1));
+        Assertions.assertTrue(Precision.equals(0, statistics.getRowCount(), 0.1));
```

Comment:
why this test changed?

Reply:
Remove some useless codes in unit tests, those codes is hard to maintaince and it's not a good idea for testing the accurracy of stats estimation according to the advise from @englefly